src/main/php/lang/ast/emit/PHP85.class.php

```diff
@@ -19,7 +19,7 @@
  * @see  https://wiki.php.net/rfc#php_85
  */
 class PHP85 extends PHP {
-  use RewriteBlockLambdaExpressions;
+  use RewriteBlockLambdaExpressions, RewriteCallableClone, RewriteCloneWith; // TODO: Remove once PR is merged!
```

Comment:
Once the PR in `php-src` is merged, we can drop these and natively emit `clone` expressions as function calls with their arguments.